src/styles/main.scss

```diff
@@ -151,17 +151,57 @@ $background-color: #f9f9f9;
       }
     }
 
-    > .slider-button {
-      background-color: white;
+    > .slider-button, > .help-container>.help-button {
+      background-color: $light-color;
       fill: $primary-color;
-      border: solid 1px lightgray;
+      border: solid 1px $light-grey;
       outline: none;
       width: 31px;
       height: 31px;
       border-radius: 2px;
       cursor: pointer;
       margin: 0;
     }
+
+    > .help-container {
+      position: relative;
+  
+      > .help-button {
+        > svg {
+          margin: 3px;
+        }
+
+        &.help-showing {
+          background-color: lighten($light-grey, 12%);
+        }
+      }
+  
+      > .help-menu {
+        background-color: $light-color;
+        border: solid 1px $light-grey;
+        position: absolute;
+        padding: 10px;
+        bottom: 0;
+        left: 31px;
```

Comment:
Is 31px connected to size of button above? (Maybe make this a variable to make the dependency more clear?)

Reply:
Thanks. Added a variable.